File: common/protcol/c#/caller/hub_call_centercaller.cs
/*this caller file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.IO;

namespace caller
{
    public class hub_call_center : juggle.Icaller 
    {
        public hub_call_center(juggle.Ichannel _ch) : base(_ch)
        {
            module_name = "hub_call_center";
        }

        public void closed()
        {
            ArrayList _argv = new ArrayList();
            call_module_method("closed", _argv);
        }

    }
}

File: common/protcol/js/caller/hub_call_centercaller.js
/*this caller file is codegen by juggle for js*/
function hub_call_center_caller(ch){
    Icaller.call(this, "hub_call_center", ch);

    this.closed = function(){
        var _argv = [];
        this.call_module_method.call(this, "closed", _argv);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Icaller.prototype;
    hub_call_center_caller.prototype = new Super();
})();
hub_call_center_caller.prototype.constructor = hub_call_center_caller;

